Gemfile.jruby-2.3.lock.release

```diff
@@ -287,9 +286,8 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       rufus-scheduler
       stud (~> 0.0.22)
-    logstash-input-file (4.0.5)
+    logstash-input-file (4.1.2)
```

Comment:
@guyboertje - do you feel comfortable enough with the passivity of this to upgrade this in a patch release ?

Reply:
let's skip this